### ISSUE TITLE
Example interop-with-html-webpack-plugin not working with html-webpack-plugin v4.3.0

### ISSUE THREAD

**Issue by igordos** [kisenka/svg-sprite-loader#406]
Code in [template.ejs](https://github.com/JetBrains/svg-sprite-loader/blob/2019115151723d98e80f14d806e2f518ee9c6191/examples/interop-with-html-webpack-plugin/template.ejs) not working with html-webpack-plugin v4.3.0, but working with v3.2.0. 
```
<% if (htmlWebpackPlugin.files.sprites) { %>
  <% for (var spriteFileName in htmlWebpackPlugin.files.sprites) { %>
    <%= htmlWebpackPlugin.files.sprites[spriteFileName] %>
  <% } %>
<% } %>
```
- Node.js version: 12.16.1
- webpack version: 4.43.0
- svg-sprite-loader version: 5.0.0 
- OS type & version: Windows 10

Can you update code for new version? 
Thank you.

**Comment by sqal** [comment]
Yeah, I ran into the same issue. As a workaround you can use this snippet to inject your SVGs instead of the old one:

```<%= compilation.assets["sprite.svg"].source() %>```

**Comment by quasarchimaere** [comment]
I am currently also running into this issue, however the approach of @sqal does not work for me since my sprite contains a contenthash (or i am missing some webpack.conf parameters), can someone help me out here?
- [webpack.config.ts](https://github.com/researchstudio-sat/webofneeds/blob/master/webofneeds/won-owner-webapp/src/main/webapp/webpack.config.ts)
- [template.ejs](https://github.com/researchstudio-sat/webofneeds/blob/master/webofneeds/won-owner-webapp/src/main/webapp/template.ejs) <- this template still contains the old interop code we used to use

**Comment by quasarchimaere** [comment]
> I am currently also running into this issue, however the approach of @sqal does not work for me since my sprite contains a contenthash (or i am missing some webpack.conf parameters), can someone help me out here?
> 
> * [webpack.config.ts](https://github.com/researchstudio-sat/webofneeds/blob/master/webofneeds/won-owner-webapp/src/main/webapp/webpack.config.ts)
> * [template.ejs](https://github.com/researchstudio-sat/webofneeds/blob/master/webofneeds/won-owner-webapp/src/main/webapp/template.ejs) <- this template still contains the old interop code we used to use

i am still searching for a working solution for this issue, is there a new way to do this now? @kisenka

**Comment by rjeny** [comment]
@quasarchimaere as I see your config you set `plainSprite: true` in SpriteLoaderPlugin, so you will only have `sprite.svg` and code below will work.

But if it`s not true. Ask yourself a question: you want to hash objets, that you want to be inline. For what?)

**Comment by quasarchimaere** [comment]
@rjeny i will try the approach you suggested and will get back to you on that, i see your point, inlining the sprite makes the hash useless. I think inlining would just be a workaround for me anyway, what i actually would have liked to achieve is to include (and preload) the hashed spritesheet in the template/index.html -> similar to the js and css resources in our project.
